source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,331 @@
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindEqPOCO.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+The following example creates a query filter using builders that matches all 
+documents where the ``make`` field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindEqBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+.. tip:: Find All Documents
+
+   Using an empty query filter matches all documents in the collection. Create
+   an empty query filter with builders as follows:
+
+   .. code-block:: csharp
+
+      var result = _guitarsCollection.Find(Builders<Guitar>.Filter.Empty).ToList();
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Comparison Operators
+--------------------
+
+Comparison operators analyze the value in a document against the specified value
+in your query filter. Common comparison operators include:
+
+.. list-table::
+   :widths: 30 30 40
+   :header-rows: 1
+
+   * - Expression
+     - Builder
+     - Description
+
+   * - ``>``
+     - ``Gt()``
+     - Greater than
+
+   * - ``<=``
+     - ``Lt()``
+     - Less than or equal to
+
+   * - ``!=``
+     - ``Ne()``
+     - Not equal to
+
+For a full list of comparison operators, see the :manual:`Comparison
+Query Operators </reference/operator/query-comparison/>` page.
+
+The following example specifies a query filter as a parameter to the ``Find()``
+method, using the ``Guitar`` class. The query matches all documents where the
+``establishedYear`` field is greater than ``1985``.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindGtPOCO.cs
+      :language:  csharp
+
+   .. output::
+      :language: json
+      :visible:
+
+      { "_id" : 4, "make" : "Kiesel", "models" : ["Ares", "Vader", "Solo"], "establishedYear" : 2015, "rating" : null }
+
+The following example uses builders to create a query filter that matches all 
+documents where the ``establishedYear`` field is greater than ``1985``.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/fundamentals/code-examples/specify-query/FindGtBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: json
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Logical Operators
+-----------------
+
+Logical operators query data using logic applied to the results of two or more
+sets of search criteria. Common logical operators include:
```

Comment:
Question/Issue: I believe "query data" should be "match documents" correct? The operator builds a filter rather than actually searches I believe.

Question/Issue: Is there a reason this section uses the term "search criteria"? I don't believe this term is defined or used outside this section.

Issue: [Avoid introducing lists with sentence fragments](https://www.mongodb.com/docs/meta/style-guide/style/lists/#introductory-text)

Issue: Avoid editorial language like "common". I believe we should instead just introduce the list as a subset of operators.

```suggestion
Logical operators are boolean expressions that match documents which evaluate to true. The following is a list of some logical operators:
```

Reply:
Match documents is a better term here - changed!

I didn't take your exact suggested change here, but tried to edit the sentence to convey the same meaning.